src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+
+  * kty: The "kty" (key type) parameter identifies the cryptographic algorithm family used with the key, such as "EC" for Elliptic Curve.
+
+  * use: The "use" (public key use) parameter identifies the intended use of the public key such as "sig" for signature.
+
+  * crv : The "crv" (Curve) parameter represents the elliptic curve-type and the hash-type such as "secp256k1" represents curve-type `secp256k1` and the hash-type `SHA-256`. 
+
+  * x : Since we assume the "alg" parameter as "ES256K" which is one from the ECDSA family, so "x" parameter represents the X-coordinate of the corresponding public key.
+
+  * y : Since we assume the "alg" parameter as "ES256K" which is one from the ECDSA family, so "y" parameter represents the Y-coordinate of the corresponding public key.
+
+Note: "jwk" is one way way of embedding public key in the JOSE header. For more details on other possible options for "alg" and representing public keys refer to [RFC7515][]. 
+
+### JWS Payload
+ The JWS payload is the message that needs to be signed.
+
+      {
+              "exp"  : 1596496501,
+              "payId": "bob$wallet.com",
+              "payIdAddress": {
+                "expTime": 34874613475,
+                "paymentNetwork": "XRPL",
+                "environment": "TESTNET",
+                "addressDetailsType": "CryptoAddressDetails",
+                "addressDetails": {
+                  "address": "rnzBSt9ZCJSh4RxC9f1v6oS9WZtEYJa8B9",
+                  "tag": "12345"
+                }
+              }
+      }
+
+#### exp
+The `exp` field is an optional field as described in [RFC7519][]. If used, it SHOULD be set to the expiration time of the cryptographic key used to generate the digital signature.
+
+#### payId
+The `payId` field is a required field. The value of `payId` field is the PayID URI in the client request that identifies the payment account information that the JSON object describes.
+
+#### PayIDAddress
+The `PayIDAddress` is a required field. The value of `PayIDAddress` field is a JSON object with the following keys: 
+
+  * "expTime": This is an optionl field and follows the same structure as described for "exp" field in [RFC7519][]. If used, the value of `expTime` SHOULD be set to the maximum time upto which the payment address in the `address` field is valid.
+
+  * "paymentNetwork": The value of the `paymentNetwork` is the value of payment-network string as specified in the client request's `Accept` header.
+
+  * "environment": The value of `environment` string is the value of environment as specified in the client request's `Accept` header.
+
+  * "addressDetailsType": The value of `addressDetailsType` is one of the following strings:
+
+       * CryptoAddressDetails
```

Comment:
Should we provide context around what these mean (or maybe point back to [PAYID-PROTOCOL]?

Reply:
Yes, we can add a reference to [PAYID-PROTOCOL]